### PR TITLE
CSS in reactJs with Bootstrap

### LINKED ISSUE #1160: Create React Styling lesson outline
add short lesson into the [react folder](https://github.com/Techtonica/curriculum/tree/master/react-js)
[template markdown](https://github.com/Techtonica/curriculum/blob/master/_templates/topic-outline.md)

### Things to Cover
* inline styles and CSS
* integrating bootstrap
* styling in JS frameworks
* create slideshow for visual learners

### PR DESCRIPTION
**Pull request by ashishnagpal2498**
fixes #1160

### CODE REVIEW

**Comment by ashishnagpal2498** [comment]
Issue Fix #1160

**Comment by alodahl** [comment]
Thanks for linking the issue, @ashishnagpal2498 .  Next time add `fixes #1160` to the description so it's easy to see. Also the wording "issue fix #1160" will not close the issue once your PR is merged, but `fixes #1160` will.

File: react-js/styling-react.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- [Techtonica sildes: CSS in ReactJs ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+
+
+### Common Mistakes & Misconceptions
+
+
+- Combining different classes based on certain state/props.
+- Wrong path provided while importing CSS.
+- CamelCase is supported in ReactJS for CSS styles.
```
Comment (alodahl) [inline react-js/styling-react.md]: Maybe have these in the lesson... why does it need to be camelcase?  What will happen if it's not? what would happen if they put "classname" or "background-image" or "font-family"?

File: react-js/styling-react.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Styling in ReactJS
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Independent Practice: 20 mins.
+
+### Prerequisites
+- CSS: [css](../css/css.md)
```
Comment (alodahl) [inline react-js/styling-react.md]: Can you title these the way their respective files are titled, and add the word "Outline" or "Lesson"?  Also, there's not need for `.` or `..`. 
example: 
-  [CSS Lesson](/css/css.md)

Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
- [CSS](/css/css.md)
```

File: react-js/styling-react.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### Lesson
+
+- [Techtonica sildes: CSS in ReactJs ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+
+
+### Common Mistakes & Misconceptions
+
+
+- Combining different classes based on certain state/props.
```
[the remark this answers is not in the document]
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: Sure.

File: react-js/styling-react.md [hunk 4]
```diff
@@ -1,90 +1,62 @@
...
+- ReactJs: [react](./react.md)
 
-Here are topics that should be understood before this topic:
-
-- Example: [Git version control](../git-version-control/git-version-control.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+Styling has been one of the building blocks of web development. Cascading Style Sheets is a style sheet language used for describing the presentation of a document written in a markup language like HTML. React only provides a virtual DOM , whereas the presentation part still lies with CSS.
```
Comment (Kundan28) [inline react-js/styling-react.md]: Remove white-space after DOM.

File: react-js/styling-react.md [hunk 5]
```diff
@@ -34,8 +34,8 @@ Styling has been one of the building blocks of web development. Cascading Style
 
 ### Lesson
 
-- [Be sure to link the slideshow, video,](https://google.com)
-- [or other materials you expect them to follow as part of the lesson.](https://google.com)
+- [Techtonica sildes: CSS in ReactJs ](https://drive.google.com/open?id=1xVsjUzPsNGq2sPaoNvc0eLLXy2Zm3kVk)
```
Comment (Kundan28) [inline react-js/styling-react.md]: @alodahl Is it necessary to mention `Techtonica sildes` ?

File: react-js/styling-react.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+- Importing stylesheets into reactJs application.
+
+### Materials
+
+- [Styling of ReactJs App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external css, internal and css modules.
+- [Bootstrap in React](https://www.youtube.com/watch?v=wdbzLi0tWq0) - Bootstrap in ReactJs can be used using CDN or by installing the package. 
+
+### Lesson
+
+- [Techtonica sildes: CSS in ReactJs ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
```
[the remark this answers is not in the document]
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: I am not able to transfer the ownership as the option is not available. Kindly guide me how to do that.
image

File: react-js/styling-react.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+
+
+### Common Mistakes & Misconceptions
+
+
+- Combining different classes based on certain state/props.
+- Wrong path provided while importing CSS.
+- CamelCase is supported in ReactJS for CSS styles.
+
```
[the remark this answers is not in the document]
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: Added

File: react-js/styling-react.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Kundan28) [reply react-js/styling-react.md]: Have a look at this https://support.google.com/drive/answer/2494892

File: react-js/styling-react.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
-- Independent Practice: 20 mins.
+- Guided Practice: 20 mins.
 
 ### Prerequisites
-- CSS: [css](../css/css.md)
-- Bootstrap: [bootstrap](../bootstrap/bootstrap.md)
-- ReactJs: [reactJs](./react.md)
+- [CSS Lesson](../css/css.md)
+- [Bootstrap Lesson](../bootstrap/bootstrap.md)
+- [React Lesson](./react.md)
```
Comment (Kundan28) [inline react-js/styling-react.md]: @alodahl Already told that Also, there's not need for `.` or `..`.

File: react-js/styling-react.md [hunk 9]
```diff
@@ -23,38 +23,80 @@ Styling has been one of the building blocks of web development. Cascading Style
...
 
 - Defining styles as objects.
-- Importing stylesheets into reactJs application.
+- Importing stylesheets into react application.
 
 ### Materials
 
-- [Styling of ReactJs App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external css, internal and css modules.
-- [Bootstrap in React](https://www.youtube.com/watch?v=wdbzLi0tWq0) - Bootstrap in ReactJs can be used using CDN or by installing the package. 
+- [Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external css, internal and css modules.
```
Comment (Kundan28) [inline react-js/styling-react.md]: css -> CSS

File: react-js/styling-react.md [hunk 10]
```diff
@@ -23,38 +23,80 @@ Styling has been one of the building blocks of web development. Cascading Style
...
 
-### Independent Practice
-
-Create simple applications by adding Css to it. Then change the css to bootstrap classes.
-
-
-### Check for Understanding
+- Wrong path provided while importing CSS.
+  
+  - If the path provided for external css file or module in the JSX file is incorrect then react throws an error i.e it will not be able to build the webpack.
```
Comment (Kundan28) [inline react-js/styling-react.md]: css -> CSS

File: react-js/styling-react.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: I saw this already , not getting option to change the owner. @Kundan28

File: react-js/styling-react.md [hunk 11]
```diff
@@ -100,4 +100,10 @@ Styling has been one of the building blocks of web development. Cascading Style
 	
 ### Supplemental Materials
 - [React Strap](https://reactstrap.github.io/) - Provides predefined components for React styled with Bootstrap.
-- [Styled Components](https://styled-components.com/docs/basics#getting-started) is the result of wondering how we could enhance CSS for styling React component systems. 
\ No newline at end of file
+  - Why to choose Reactstrap ?
+    - Directly use components instead of  bootstrap typing classes and passing attributes as props.
+	- Closely relate bootstrap to react.
```
Comment (Kundan28) [inline react-js/styling-react.md]: react -> React

File: react-js/styling-react.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+### Lesson
+
+- [Slides: CSS in React ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+### Guided Practice
+> #### Creating a simple div with InLine and External CSS and converting it into Bootstrap.
+
+1. Inline And External CSS
+  - App.js
+    - Creating a simple div using and add custom styling to it.
```
Comment (alodahl) [inline react-js/styling-react.md]: Can you explain a little more about what’s happening? Instead of “creating...”, say something like”here’s an example of...” Which one is inline, and which one is external? https://medium.com/@oleg008/inline-styles-vs-css-f8c1c926bb63
Maybe use the class name, like calling it parent or parent div and having the inner one be Child? Plus those terms are more dev-oriented than inner and outer. It will be total beginners reading this, so we want to take every opportunity to be clear.

File: react-js/styling-react.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+       }
+      .my-div div
+	  {
+	   flex-basis: 25%;
+	   max-width: 25%;
+	   padding: 10px;
+      }
+      ```
+2. Using Bootstrap
+  - App.js
```
Comment (alodahl) [inline react-js/styling-react.md]: Please explain how this one is different. What should the apprentice notice?

File: react-js/styling-react.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+	    <div className="row mt-2 mb-2" style={{backgroundColor:"darkorange"}}>
+		<div className="col-3 p-2" style={{backgroundColor:"red"}}>Inner Div 1</div>
+		<div className="col-3 p-2" style={{backgroundColor:"blue"}}>Inner Div 2</div>
+		<div className="col-3 p-2" style={{backgroundColor:"green"}}>Inner Div 3</div>
+		</div>
+      ```
+
+
+### Common Mistakes & Misconceptions
+
```
Comment (alodahl) [inline react-js/styling-react.md]: Wait, is this a bad thing? I have to admit I don’t understand this point, can you reword it? What would be the wrong thing to do? What should they do instead?

File: react-js/styling-react.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+	<button className = {this.props.color ? "error" : "" + "btn-class" }>ButtonComp </button>
+  ```
+  - If color is received as prop by the component then **className** will be 
+    - > className = "error btn-class" 
+  - else
+    - > className = "btn-class"
+
+- Wrong path provided while importing CSS.
+  
+  - If the path provided for external CSS file or module in the JSX file is incorrect then react throws an error i.e it will not be able to build the webpack.
```
Comment (alodahl) [inline react-js/styling-react.md]: What does the error look like?

File: react-js/styling-react.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+
+- CamelCase is supported in ReactJS for CSS styles.
+	- using **class** instead of **className** gives warning.
+	- error encounters if any inline CSS property is used as **two words** like **background-color** because inline CSS is treated as an object in react.
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
	- If any inline style property is used as **two words** like **background-color**, you will encounter an error.  This is because these styles are not in a .css file. They are in JSX (Javascript extension for React), so it is actually a JS object; and JS cannot parse dashes. Also note the double curly-braces. The outer set of braces means “now instead of React JSX, what’s in these braces will be pure JavaScript.”  style={}. Then inside is a JS object {backgroundColor: “red”}.
```

File: react-js/styling-react.md [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+    - > className = "error btn-class" 
+  - else
+    - > className = "btn-class"
+
+- Wrong path provided while importing CSS.
+  
+  - If the path provided for external CSS file or module in the JSX file is incorrect then react throws an error i.e it will not be able to build the webpack.
+
+- CamelCase is supported in ReactJS for CSS styles.
+	- using **class** instead of **className** gives warning.
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
	- Using **class** instead of **className** causes a console warning.
```

File: react-js/styling-react.md [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+	- error encounters if any inline CSS property is used as **two words** like **background-color** because inline CSS is treated as an object in react.
+	- > Incorrect Way - ``` <button style={{background-color:"red" }}> ```
```
Comment (alodahl) [inline react-js/styling-react.md]: 👍

File: react-js/styling-react.md [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+	- > Correct Way   ``` <button style={{backgroundColor:"red"}} > ```
+
+	
+### Supplemental Materials
+- [React Strap](https://reactstrap.github.io/) - Provides predefined components for React styled with Bootstrap.
+  - Why to choose Reactstrap ?
+    - Directly use components instead of  bootstrap typing classes and passing attributes as props.
```
Comment (alodahl) [inline react-js/styling-react.md]: Please put these lines in a syntax that uses “With Reactstrap, you can...” or “It allows you to...”

File: react-js/styling-react.md [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+
+- [Video: Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external CSS, internal and CSS modules.
+- [Video: Bootstrap in React](https://www.youtube.com/watch?v=wdbzLi0tWq0) - Bootstrap in React can be used using CDN or by installing the package. 
+
+### Lesson
+
+- [Slides: CSS in React ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+### Guided Practice
+> #### Creating a simple div with InLine and External CSS and converting it into Bootstrap.
```
Comment (alodahl) [inline react-js/styling-react.md]: Are you telling them they should create this themselves right now? Add step-by-step setup directions. Should they touch a new file or a replit? Are you expecting them to start a new create-react-app? Should they run it at any point?

File: react-js/styling-react.md [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+  - App.js
+    - Creating a simple div using and add custom styling to it.
+    - ```
+      <div className="my-div">
+		<div style={{backgroundColor:"red"}}>Inner Div 1</div>
+		<div style={{backgroundColor:"blue"}}>Inner Div 2</div>
+		<div style={{backgroundColor:"green"}}>Inner Div 3</div>
+	  </div>
+      ```
+  - CSS
```
Comment (alodahl) [inline react-js/styling-react.md]: Where is this? Should they make a new file? What should they call it? How do they link it? Please tell them clearly, this is Guided Practice.

File: react-js/styling-react.md (hunk 13 above)
Comment (alodahl) [inline react-js/styling-react.md]: Please tell them what they are doing next and why. “Using bootstrap” isn’t enough. How will they know that it worked? Should they change the same file or make a new one?

File: react-js/styling-react.md (hunk 15 above)
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: The image of the error is attached in the slides.

**Comment by alodahl** [comment]
Hi @ashishnagpal2498, will you have time to make these changes?

**Comment by ashishnagpal2498** [comment]
Hi @alodahl . Yes I'll be working on the changes on weekend. I would be really glad , if you could please upgrade the PR status to "hard". Thank you.

**Comment by alodahl** [comment]
Thanks @ashishnagpal2498 !  It’s it is hard to write new documentation. However, this is about half the length of our typical outlines, since it is a short topic, so I would rank it with other medium issues we have.

File: react-js/styling-react.md [hunk 22]
```diff
@@ -0,0 +1,238 @@
...
+> #### Let's create a simple div of flex box using InLine CSS then we will achieve same using External CSS and Bootstrap separately.
+
+- First thing we need to do is create a react application.
+	- Create a react app using create react app command. `npx create-react-app`.
+    - This will create a simple react app , and now we will start making changes to this app.
+	- First of all get rid of all the unnecessary stuff showing on DOM , remove all the lines in render of `App.js`. 
+	- Now we create a simple div **parent div** which will contain few div as *child* elements.
+	- After creating that , your ``App.js`` will look like this 
+	- ```
+		import React from 'react';
```
Comment (alodahl) [inline react-js/styling-react.md]: This is great!  Please fix the indentation in the example.

File: react-js/styling-react.md [hunk 23]
```diff
@@ -0,0 +1,238 @@
...
+    	<div className="App">
+      	<div> {/* Start Parent Div */}
+	  	{/* Child Div */}
+		<div>Child Div 1</div>
+		<div>Child Div 2</div>
+		<div>Child Div 3</div>
+	  	</div>
+	 	{/* Child Div */}
+		</div>
+		/* End of Parent Div */
```
Comment (alodahl) [inline react-js/styling-react.md]: Doesn't this need brackets too?
```suggestion
		{/* End of Parent Div */}
```

File: react-js/styling-react.md [hunk 24]
```diff
@@ -0,0 +1,238 @@
...
+- [Video: Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external CSS, internal and CSS modules.
+- [Video: Bootstrap in React](https://www.youtube.com/watch?v=wdbzLi0tWq0) - Bootstrap in React can be used using CDN or by installing the package. 
+
+### Lesson
+
+- [Slides: CSS in React ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+### Guided Practice
+
+> #### Let's create a simple div of flex box using InLine CSS then we will achieve same using External CSS and Bootstrap separately.
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
> #### Let's create a simple div, and style it using inline CSS and flexbox.  Then we will achieve same using External CSS and Bootstrap separately.
```
"we will achieve same using External CSS and Bootstrap separately." Can you rephrase this? Do you mean External CSS and Bootstrap separately from the inline one above, or are we doing 3 things: inline, then external, and last Bootstrap?

File: react-js/styling-react.md (hunk 23 above)
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: Actually , I wanted to show the comments to make it clear. So ya brackets are necessary in react.

File: react-js/styling-react.md (hunk 22 above)
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: Okay sure.

File: react-js/styling-react.md (hunk 24 above)
Comment (ashishnagpal2498) [reply react-js/styling-react.md]: We are doing same thing in 3 different ways using , Inline CSS then by external CSS and finally using Bootstrap.

**Comment by alodahl** [comment]
image

File: react-js/styling-react.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (alodahl) [reply react-js/styling-react.md]: This is likely because it is a pptx and not a google slides doc.  Please transfer it to a google slides doc and then transfer ownership.

[the remark this answers is not in the document]
Comment (alodahl) [reply react-js/styling-react.md]: It would be nice to have edit access to the pptx, though.

File: react-js/styling-react.md [hunk 25]
```diff
@@ -0,0 +1,250 @@
+# Styling in React
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Guided Practice: 20 mins.
+
+### Prerequisites
+- [CSS Lesson](/css/css)
```
Comment (alodahl) [inline react-js/styling-react.md]: These three are broken. 1. The files should probably be named .md. 2. search for the file name, such as css.md, to see if the path was changed.  I think css and bootstrap are now in the /web/ folder.

File: react-js/styling-react.md [hunk 26]
```diff
@@ -0,0 +1,250 @@
...
+- [React Lesson](react)
+
+
+### Motivation
+
+Styling has been one of the building blocks of web development. Cascading Style Sheets is a style sheet language used for describing the presentation of a document written in a markup language like HTML. React only provides a virtual DOM, whereas the presentation part still lies with CSS.
+
+### Objectives
+
+- Styling in JS Framework.
```
Comment (alodahl) [inline react-js/styling-react.md]: framework?  do you mean JSX?

File: react-js/styling-react.md [hunk 27]
```diff
@@ -0,0 +1,250 @@
...
+- Inline styling and using external Stylesheet.
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
- Inline styling and using external stylesheets.
```

File: react-js/styling-react.md [hunk 28]
```diff
@@ -0,0 +1,250 @@
...
+- Bootstrap in React.	 
+
+### Specific Things To Learn
+
+- Defining styles as objects.
+- Importing stylesheets into react application.
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
- Importing stylesheets into React application.
```

Please make sure all proper names like React and Bootstrap are capitalized, and non-proper nouns like 'inline' are not.

File: react-js/styling-react.md [hunk 29]
```diff
@@ -0,0 +1,250 @@
...
+	  		</div>
+			{{/* End of Parent Div */}}  
+		</div>
+      ```
+2. Using External CSS 
+    1. Create an external CSS file and add a parent class with same styling as you did in Inline CSS.
+	2. Create a class for each **child element**.
+	3. Your CSS file will look like -
+		- ```
+       		.parent {
```
Comment (alodahl) [inline react-js/styling-react.md]: please look at the markdown preview - all of these code samples' indentation is a bit weird.

Comment (alodahl) [reply react-js/styling-react.md]: image

File: react-js/styling-react.md [hunk 30]
```diff
@@ -0,0 +1,250 @@
...
+
+### Lesson
+
+- [Slides: CSS in React ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+### Guided Practice
+
+> #### Let's create a simple div, and style it using inline CSS and flexbox.  Then we will achieve same using External CSS and Bootstrap separately.
+
+- First thing we need to do is create a react application.
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
- First, we need to create a React application.
```

Please capitalize all instances of `React`

File: react-js/styling-react.md [hunk 31]
```diff
@@ -0,0 +1,250 @@
...
+- Bootstrap in React.	 
+
+### Specific Things To Learn
+
+- Defining styles as objects.
+- Importing stylesheets into react application.
+
+### Materials
+
+- [Video: Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external CSS, internal and CSS modules.
```
Comment (alodahl) [inline react-js/styling-react.md]: are we using "CSS Modules" anywhere now?  I don't see it.

Comment (ashishnagpal2498) [reply react-js/styling-react.md]: Yes it is similar to External CSS. But it is important to know, in case of multiple CSS file.

**Comment by ashishnagpal2498** [comment]
Hi @alodahl, I have made the changes you have requested. I am unable to transfer the ownership as there is not option of owner in that drop down menu. I have attached the screenshot before. Kindly refer to that and guide me how to do that.

File: react-js/styling-react.md (hunk 28 above)
Comment (alodahl) [reply react-js/styling-react.md]: I am not sure why you resolved this, as you didn't make the change.

File: react-js/styling-react.md [hunk 32]
```diff
@@ -0,0 +1,252 @@
...
+  		  return (
+    		<div className="App">
+      		{/* Start Parent Div */}
+			  <div>
+	  		    {/* Start of Child Divs */}
+				<div>Child Div 1</div>
+				<div>Child Div 2</div>
+				<div>Child Div 3</div>
+				{/* End of Child Divs */}
+	  			</div>
```
Comment (alodahl) [inline react-js/styling-react.md]: Don't line 61 and 63 need to line up with line 53 and 55?

File: react-js/styling-react.md [hunk 33]
```diff
@@ -0,0 +1,252 @@
...
+			 <div className="col-3 p-2" style={{backgroundColor:"red"}}>Child Div 2</div>
+			 <div className="col-3 p-2" style={{backgroundColor:"blue"}}>Child Div 3</div>
+			 {/* End of Child Divs */}
+		</div>
+		{/* End of Parent Div */}
+	  </div>
+      ```
+
+
+### Common Mistakes and their Correction
```
Comment (alodahl) [inline react-js/styling-react.md]:
```suggestion
### Common Mistakes and Misconceptions
```

File: react-js/styling-react.md [hunk 34]
```diff
@@ -0,0 +1,252 @@
...
+
+- Combining different classes based on certain state/props.
+  - When a component is receiving a prop and depending upon the value of that prop the CSS is rendered.
```
Comment (alodahl) [inline react-js/styling-react.md]: Can you make it clear whether the following example is a mistake, or the correct way to do it?  You've already made it clear for "Wrong path" and "Camelcase", but not this one.

Comment (alodahl) [reply react-js/styling-react.md]: Just this and transferring the slides, and you’re done ✅

Comment (alodahl) [reply react-js/styling-react.md]: I changed the label to hard, as I appreciate how much work you put into the slides.

Comment (ashishnagpal2498) [reply react-js/styling-react.md]: Thank you @alodahl , I have transferred the ownership of slides and also covered this in the new commit.

File: react-js/styling-react.md (hunk 28 above)
Comment (alodahl) [reply react-js/styling-react.md]: Why do you keep resolving this, it’s important.
